packages/analytics-element/src/example-component.js

```diff
@@ -0,0 +1,47 @@
+import { LitElement, html, css } from 'lit-element';
+import TrackedElement from './tracked-element';
+
+class ExampleComponent extends LitElement {
+  static get styles() {
+    return css`
+      fieldset {
+        padding: 0;
+        border: 0;
+      }
+      button {
+        display: inline-block;
+        width: 320px;
+        height: 40px;
+        padding: 10px 24px;
+        font-size: 16px;
+        line-height: 16px;
+        box-sizing: border-box;
+      }
+    `;
+  }
+
+  click(e) {
+    this.trackClick(e);
+  }
+
+  submit(e) {
+    this.trackSubmit(e);
+    e.preventDefault();
+  }
+
+  render() {
+    return html`
+      <h1>Click and Submit Tracking</h1>
+      <p>View the console for event logging example</p>
+      <form action="" method="GET" @submit=${this.submit} data-event-submit-tracking="ExampleTracker|Submit">
+        <fieldset>
+          <button @click=${this.click} data-event-click-tracking="ExampleTracker|Click">Track click</button
+        </fieldset>
+      </form>
+    `;
+  }
+}
+
+Object.assign(ExampleComponent.prototype, TrackedElement);
```

Comment:
I don't think we should bind trackers to the object.  They are pure functions, and don't need/modify state, can they just be used ad-hoc in the code?
Perhaps we need to set a flag instead to show in constructor that we are tracking these links? `this._sendsEvents` or something..  I really like immediately knowing if this component has a special side effect like this, (taking a page from @cdrini and his preference of immediate declarations).